plugins/woo-ai/includes/class-woo-ai-settings.php

```diff
@@ -0,0 +1,220 @@
+			);
+		}
+
+		return $settings;
+	}
+
+	/**
+	 * Return array describing new settings.
+	 *
+	 * @return array Describing new AI settings.
+	 */
+	public function get_woo_ai_settings() {
+
+		// These are made available in the DOM and actually toggled client-side.
+		$tone_desc = array(
+			'informal'     => __( 'Relaxed and friendly, as if you were having a conversation with a friend.', 'woocommerce' ),
+			'humorous'     => __( 'Engage customers with a light-hearted and fun communication style.', 'woocommerce' ),
+			'neutral'      => __( 'A balanced and impartial tone that uses casual expressions without slang.', 'woocommerce' ),
+			'youthful'     => __( 'Bring energy and enthusiasm to your store with a friendly and cheeky tone.', 'woocommerce' ),
+			'formal'       => __( 'Direct yet respectful, a formal tone sounds serious and professional.', 'woocommerce' ),
+			'motivational' => __( 'Passionate and inspiring, engage users with this upbeat tone.', 'woocommerce' ),
+		);
+
+		$markup_str = array_reduce(
+			array_keys( $tone_desc ),
+			function( $acc, $key ) use ( $tone_desc ) {
+				$value = $tone_desc[ $key ];
+				return $acc . '<span class="woo-ai-settings-tone-option" data-option="' . htmlspecialchars( $key ) . '">' . htmlspecialchars( $value ) . '</span>';
+			},
+			''
+		);
+
+		return array(
+			array(
+				'id'    => 'woo_ai_title',
+				'type'  => 'title',
+				'title' => __( 'Artificial Intelligence', 'woocommerce' ),
+				'desc'  => __( "Save time by automating mundane parts of store management. This information will make AI-generated content, visuals, and setting more aligned with your store's goals and identity.", 'woocommerce' ),
+			),
+
+			array(
+				'title'   => __( 'Enable AI', 'woocommerce' ),
+				'desc'    => __( 'Enable AI features in your store', 'woocommerce' ),
+				'id'      => 'woo_ai_enable_checkbox',
+				'default' => 'yes',
+				'type'    => 'checkbox',
+			),
+
+			array(
+				'title'             => __( 'Tone of voice', 'woocommerce' ),
+				'id'                => 'woo_ai_tone_of_voice_select',
+				'css'               => 'min-width:300px;',
+				'default'           => 'informal',
+				'type'              => 'select',
+				'desc'              => $markup_str,
+				'desc_tip'          => __( 'Choose the language style that best resonates with your customers. It\'ll be used in text-based content, like product descriptions.', 'woocommerce' ),
+				'custom_attributes' => array( 'disabled' => 'true' ),
+				'options'           => array(
+					'informal'     => esc_html__( 'Informal', 'woocommerce' ),
+					'humorous'     => esc_html__( 'Humorous', 'woocommerce' ),
+					'neutral'      => esc_html__( 'Neutral', 'woocommerce' ),
+					'youthful'     => esc_html__( 'Youthful', 'woocommerce' ),
+					'formal'       => esc_html__( 'Formal', 'woocommerce' ),
+					'motivational' => esc_html__( 'Motivational', 'woocommerce' ),
+				),
+			),
+
+			array(
+				'id'                => 'ai_describe_store_description',
```

Comment:
should this be prefixed with `woo_ai` like it is for `woo_ai_tone_of_voice_select`?

Reply:
Sure, makes sense 👍